shlr/capstone.sh

```diff
@@ -20,11 +14,28 @@ else
 	else
 		HEAD="`git log|head -n1 | awk '{print $2}'`"
 	fi
+
 	if [ "${HEAD}" = "${CS_TIP}" ]; then
 		echo "[capstone] Already in TIP, no need to update from git"
 		exit 0
 	fi
-		echo "[capstone] Updating capstone from git..."
+}
+
+if [ -d capstone -a ! -d capstone/.git ]; then
+	echo "[capstone] release with no git?"
+	cd capstone
+	for PATCH in ../capstone-patches/* ; do
+		yes n | patch -Rp1 < $PATCH
+	done
+else
+	clone_capstone
```

Comment:
Why call it twice then?

Reply:
you are right... I just added a commit, to reset the capstone directory only if the branch is different from the expected one